packages/nimble-components/src/select/index.ts

```diff
@@ -644,7 +644,11 @@ export class Select extends FormAssociatedSelect implements ErrorPattern {
     }
 
     protected override focusAndScrollOptionIntoView(): void {
-        super.focusAndScrollOptionIntoView();
+        const optionToFocus = this.firstSelectedOption;
+        if (this.contains(document.activeElement) && optionToFocus !== null) {
```

Comment:
Can all of this code get moved into the `this.open` if-statement? Is there a reason to try to focus and scroll an item into view when the element isn't even open?

Reply:
I'm not sure if it has an effect when `!this.open`. If I move it inside the `if`, the listbox seems to still be scrolled properly when I subsequently open the drop-down, so maybe it's okay. I'm a little hesitant to introduce another change to the pre-existing behavior without knowing that it doesn't have other side-effects, though.

Reply:
After our conversation at stand-up, I'm convinced that it's better to just leave the original `super` call but move it inside the `this.open` if statement, like you suggested.